### PR TITLE
Pip install fix

### PR DESCRIPTION
**Pull request by Chilipp**
- [x] Closes https://github.com/Chilipp/straditize/issues/1, https://github.com/Chilipp/straditize/issues/8
 - [x] Tests passed (for all non-documentation changes)

### CODE REVIEW

**Comment by Chilipp** [comment]
This PR is ready to be merged. It addresses https://github.com/Chilipp/straditize/issues/1 and https://github.com/Chilipp/straditize/issues/8. In particular:

- it adds pyqt5 to the dependencies in setup.py (see https://github.com/Chilipp/straditize/issues/1)
- it adds tests for the installation with pip in .travis.yml (see https://github.com/Chilipp/straditize/pull/6/commits/a561cba26320a66c8e103ee42c31ad5ea5ce5dab and https://github.com/Chilipp/straditize/pull/6/commits/892faba99e737e9392bdca8694bb757cc6e014a2)
- it changed the installation instructions for the installation from source (see the [docs](https://straditize.readthedocs.io/en/pip-install-fix/installing.html#installation-from-source) and [README](https://github.com/Chilipp/straditize/tree/pip-install-fix#installation) and the installation with conda
- it resolves several memory leaks in the test suite (although there are still some) and improves the instructions on running the tests (see the [docs](https://straditize.readthedocs.io/en/pip-install-fix/installing.html#running-the-tests))

It will be merged if @sgrieve and @ixjlyons are okay with the proposed changes (see https://github.com/Chilipp/straditize/issues/1#issuecomment-462475855 and https://github.com/Chilipp/straditize/issues/8).

File: README.rst [hunk 1]
```diff
@@ -80,24 +80,25 @@ been used to create the diagram.
 
 Installation
 ------------
-We highly recommend to use anaconda_ and install straditize::
+We highly recommend to use anaconda_ and install straditize into an own environment::
```
Comment (ixjlyons) [inline README.rst]: I'd change `an own` to `its own`

File: README.rst [hunk 2]
```diff
@@ -80,24 +80,25 @@ been used to create the diagram.
...
+    pip install .  # or python setup.py install, but pip is recommended
 
 It can then be started from the command line via::
 
     straditize
 
 A more detailed description is provided in the docs_.
 
-.. _anaconda: https://www.continuum.io/downloads
+    .. _anaconda: https://conda.io/en/latest/miniconda.html
```
Comment (ixjlyons) [inline README.rst]: Not sure if the indentation here matters

File: docs/installing.rst [hunk 3]
```diff
@@ -24,19 +24,18 @@ default conda-channels via::
 
     $ conda config --add channels conda-forge
 
-Then you can choose. Either you install straditize into your base environment
-via::
-
-    $ conda install -c chilipp straditize
-
-or you create a new virtual environment via::
+Then you can choose. We recommend to install straditize into it's own environment via::
```
Comment (ixjlyons) [inline docs/installing.rst]: Should be `its` (without the apostrophe)

File: README.rst (hunk 1 above)
Comment (Chilipp) [reply README.rst]: thanks! resolved by https://github.com/Chilipp/straditize/pull/6/commits/5522234bca9dbad38072e85769addc84145a7096

File: README.rst (hunk 2 above)
Comment (Chilipp) [reply README.rst]: Thanks! This is resolved by https://github.com/Chilipp/straditize/pull/6/commits/5522234bca9dbad38072e85769addc84145a7096

File: docs/installing.rst (hunk 3 above)
Comment (Chilipp) [reply docs/installing.rst]: Thanks! I resolved this by https://github.com/Chilipp/straditize/pull/6/commits/0e9f8af8db813470ded6c0af607b8fbf628a50b9

**Comment by Chilipp** [comment]
Thanks a lot for your quick response and your time! The suggested changes have been implemented. I will merge this PR now.

**Pull request merged**